src/components/sbb-train/sbb-train.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component, h, JSX, Prop } from '@stencil/core';
+import { InterfaceSbbTrainAttributes } from './sbb-train.custom.d';
+
+/**
+ * @slot unnamed - Use this to document a slot.
+ */
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-train.scss',
+  tag: 'sbb-train',
+})
+export class SbbTrain {
+  // TODO Accessibility text for driving direction and adjust readme
+
+  /** General label for "driving direction" */
```

Comment:
Most of the time, we add add a dot to every description at the end. I only write this here, but in all component in this PR it could be standardized.